sdk/tables/aztable/table_client.go

```diff
@@ -53,19 +53,20 @@ func (t *TableClient) Delete(ctx context.Context) (TableDeleteResponse, error) {
 // Top: The maximum number of entities that will be returned per page of results.
 // Note: This value does not limit the total number of results if NextPage is called on the returned Pager until it returns false.
 //
-// Query returns a Pager, which allows iteration through each page of results. Example:
+// List returns a Pager, which allows iteration through each page of results. Example:
 //
-// pager := client.Query(nil)
+// options := &ListOptions{Filter: to.StringPtr("PartitionKey eq 'pk001'"), Top: to.Int32Ptr(25), Select: to.StringPtr("PartitionKey,RowKey,Value,Price")}
```

Comment:
I think PK and RK are always returned no matter what and so you may want to document this and NOT show them in the doc comment.

Reply:
I dont believe this is true, [here](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/tables/azure-data-tables/tests/recordings/test_table_entity.test_get_entity_with_select.yaml#L108-L153) is a recording for the python suite that shows a response without the pk/rk. For reference, [here](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/tables/azure-data-tables/tests/test_table_entity.py#L618-L636) is the test as well.

Reply:
OK, maybe I'm wrong. See the request here where they are selecting the PK & RK: https://docs.microsoft.com/en-us/rest/api/storageservices/query-entities#sample-request